shell/edit/provisioning.cattle.io.cluster/__tests__/Basics.test.ts

```diff
@@ -264,11 +264,11 @@ describe('component: Basics', () => {
     expect((wrapper.vm as unknown as any).psaOptions[0].label).toStrictEqual(`${ label } (Current)`);
   });
 
-  it.skip.each([
-    ['anything', false, true], // TODO: This first option doesn't appear to be working correctly. An override of false will always evaluate to false
+  it.each([
+    ['anything', false, false],
```

Comment:
It seems this test case was wrong per Basics.vue inline comments "Disable PSA if CIS hardening is enabled, except override"

Reply:
I think we could remove the third case if not necessary, or what's your opinion?

Reply:
Agreed, it's gone now.